static/gsAdmin/components/customers/customerOverview.tsx

```diff
@@ -638,7 +638,9 @@ function CustomerOverview({customer, onAction, organization}: Props) {
           <DetailLabel title="Internal ID">{customer.id}</DetailLabel>
           <DetailLabel title="Data Storage Location">{region}</DetailLabel>
           <DetailLabel title="Data Retention">
-            {customer.dataRetention || '90d'}
+            {customer.orgRetention?.standard
+              ? `${customer.orgRetention?.standard}d`
+              : '90d'}
```

Comment:
what's the reason we fallback to 90d?

Reply:
Good catch. This is actually an existing bug, because we show 90-day retention for free orgs.
It should fall back to the errors retention for the org (all plans have `DataCategory.ERROR`), and then perhaps fall back to 90.